File: src/main/scala/io/github/h2sxxa/scala3std/forge/Scala3Std.scala
package io.github.h2sxxa.scala3std.forge

import cross.net.minecraftforge.fml.common.Mod
import io.github.h2sxxa.scala3std.common.LibDetector

import scala.annotation.unused
import scala.util.Properties

@unused
@Mod(
  value = "scala3std",
  modid = "scala3std",
  name = "Scala3Std",
  version = "3.5.1+C",
  modLanguage = "scala",
)
object Scala3Std {
  LibDetector.scan()
}

File: src/main/java/io/github/h2sxxa/scala3std/forge/Scala3Std.java
package io.github.h2sxxa.scala3std.forge;

import cross.net.minecraftforge.fml.common.Mod;
import io.github.h2sxxa.scala3std.ModInfo;
import io.github.h2sxxa.scala3std.common.LibDetector;
import scala.collection.immutable.List;

import java.util.Properties;

@SuppressWarnings("unused")
@Mod(value = ModInfo.MODID)
public class Scala3Std {
    public static final List<Properties> properties = LibDetector.scan();
}
